syft/frameworks/torch/tensors/interpreters/replicated_shared.py

```diff
@@ -270,6 +273,98 @@ def shape(self):
     def players(self):
         return self.__get_players()
 
+    def set_garbage_collect_data(self, value):
+        shares = self.child
+        for _, share in shares.items():
+            share[0].garbage_collect_data = value
+            share[1].garbage_collect_data = value
+
+    def get_garbage_collect_data(self):
+        shares = self.child
+        res = {}
+
+        for worker, shares in shares.items():
+            assert shares[0].garbage_collect_data == shares[1].garbage_collect_data
+            res[worker.id] = shares[0].garbage_collect_data
+
+        return res
+
+    @property
+    def grad(self):
+        """
+        Gradient makes no sense for Additive Shared Tensor, so we make it clear
+        that if someone query .grad on a Additive Shared Tensor it doesn't error
```

Comment:
```suggestion
        Gradient makes no sense for Replicated Shared Tensor, so we make it clear
        that if someone query .grad on a Replicated Shared Tensor it doesn't error
```

Reply:
Alternatively you can maybe import AdditiveSharingTensor and just copy the method?
`grad = AdditiveSharingTensor.grad` ?

Reply:
Hmmm...I am inclined to keep them separated such that each RST is not coupled with the AST (even if the RST is an AST behind)